jekyll/_cci2/custom-images.md

```diff
@@ -20,6 +20,10 @@ Watch the following video for a detailed tutorial of customizing Docker images.
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/JYVLeguIbe0" frameborder="0" allowfullscreen></iframe>
 
+## How to Use the CircleCI Dockerfile Wizard
```

Comment:
This seems a little misleading. Rather than showing *how* to use the CircleCI Dockerfile Wizard, we're just linking and telling users to follow the instructions there.

It feels like the wizard is another (recommended?) method of creating a custom image. So maybe we write something like: "There are two ways to create a custom image: with our Dockferfile Wizard or manually."

Reply:
Right, I will qualify the heading like we do for the video heading